packages/suite/src/views/onboarding/index.tsx

```diff
@@ -58,6 +66,7 @@ const Onboarding = ({ prerequisite }: InjectedModalApplicationProps) => {
 
     return (
         <LayoutComponent>
+            {useShowModal() ? modal : null /* is it ok? */}
```

Comment:
Backup and pin components have `Skip modal`, try to inspire there how they did it.

Reply:
The Skip modal is rendered differently, so I prefer to keep it this way. However, already present guide/modal interference in onboarding must be fixed for this modal as well as part of #4755.